demos/jans-tarp/src/options/cedarling.tsx

```diff
@@ -355,11 +357,11 @@ export default function CedarlingMgmt({ data, notifyOnDataChange, isOidcClientRe
                                             </Accordion> : ''}
                                     </Box>}
                             </>}
+                            <Box style={{textAlign: 'right', width: "60%"}}>© Gluu Inc. All Rights Reserved.</Box>
```

Comment:
This is a Janssen Project repo... it should be It would be better to have "Janssen Project" there and that's it.